src/loops/for-loop-basic.js

```diff
@@ -4,12 +4,26 @@ const evenNums = []
 const countdown = []
 
 // TODO: Write a for loop that adds the numbers 0 to 3 to the numsZeroToThree array
+for (let i = 0; i < 4; i++) {
+  numsZeroToThree.push(i)
+}
 
 // TODO: Write a for loop that adds the numbers 5 to 10 to the numsFiveToTen array
+for (let i = 5; i < 11; i++) {
+  numsFiveToTen.push(i)
+}
 
 // TODO: Write a for loop that adds all the even numbers between 0 and 6 (0, 2, 4, 6) to evenNums
+const numbers = [0, 2, 4, 6];
+for (let i = 0; i < numbers.length; i++) {
+  if (numbers[i]) 2 === 0;
```

Comment:
this if statement won't work: you will need to place all the comparison instruction inside the ():

`if(numbers[i] % 2 === 0) {`